MVC/application/view/components/profileForm.php

```diff
@@ -0,0 +1,62 @@
+<form action="http://mvc-task.com/userControl/editUserProfile" method="post" enctype="multipart/form-data">
+    <dl>
+      <dd class="dd-img">
+        <img src="<?php echo $_SESSION['userImageAddress']; ?>" alt="" width="100" height="100">
+      </dd>
+      <dt><label for="first_name">Your full name</label></dt>
+      <dd>
+        <input type="text" name="first_name" id="first_name" required onblur="checkFname()" placeholder="Enter your first name"
+        value="<?php if(isset($_SESSION['userFirstName']) && isset($_SESSION['userLastName'])){echo $_SESSION['userFirstName'] ." ". $_SESSION['userLastName'];} ?>"
+        >
+      </dd>
+      <dd>
+        <span id="invalid_fname"></span>
+      </dd>
+      <!-- <dt><label for="last_name">Enter your last name</label></dt>
```

Comment:
Please remove the following line of codes if they are not being used

Reply:
In this file I'll update new things that's why it is their.